spec/tbd_tests_spec.rb

```diff
@@ -1164,6 +1164,55 @@
     model.save(file, true)
   end
 
+  it "can generate and access KIVA inputs (midrise apts)" do
+    translator = OpenStudio::OSVersion::VersionTranslator.new
+    TBD.clean!
+
+    argh  = {}
+    file  = File.join(__dir__, "files/osms/in/midrise.osm")
```

Comment:
Tests KIVA inputs, but also tests for zone multipliers (adiabatic conditions for top-story floors), as discussed [here](https://github.com/rd2/tbd/pull/109/commits/d87c4c7a0d227f36e762aa0d636897d9750dbb5e#r1319762636).